### ISSUE TITLE
Question about FactorizedReduce

### ISSUE THREAD

**Issue by pprp** [microsoft/nni#3170]
Hi, I really appreciate your excellent work. I am currently studying ENAS and I am confused about the FactorizedReduce function which located at https://github.com/microsoft/nni/blob/master/examples/nas/enas/ops.py

```
class FactorizedReduce(nn.Module):
    def __init__(self, C_in, C_out, affine=False):
        super().__init__()
        self.conv1 = nn.Conv2d(C_in, C_out // 2, 1,
                               stride=2, padding=0, bias=False)
        self.conv2 = nn.Conv2d(C_in, C_out // 2, 1,
                               stride=2, padding=0, bias=False)
        self.bn = nn.BatchNorm2d(C_out, affine=affine)

    def forward(self, x):
        out = torch.cat([self.conv1(x), self.conv2(x[:, :, 1:, 1:])], dim=1)
        out = self.bn(out)
        return out
```

out = torch.cat([self.conv1(x), self.conv2(x[:, :, 1:, 1:])], dim=1) 

The above line of code is really confusing.  Can you explain why you did that? Which paper did you refer to?

**Comment by ultmaster** [comment]
https://github.com/khanrc/pt.darts/blob/48e71375c88772daac376829fb4bfebc4fb78144/models/ops.py#L165
